src/commonControls/listControl/ListControlAPL.ts

```diff
@@ -115,3 +116,234 @@ export namespace ListControlAPLPropsBuiltIns {
         };
     }
 }
+
+export type ListStyles = 'touchForward' | 'imageList';
```

Comment:
I think rename touchForward -> textList.

Reply:
Renamed in latest commit